xapian-letor/tests/api_letor.cc

```diff
@@ -81,6 +105,54 @@ db_index_two_documents(Xapian::WritableDatabase& db, const string&)
     db.add_document(doc);
 }
 
+// To check for three documents. out of which one is irrelevent
+static void
+db_index_three_documents(Xapian::WritableDatabase& db, const string&)
+{
+    Xapian::Document doc;
+    Xapian::TermGenerator termgenerator;
+    termgenerator.set_document(doc);
+    termgenerator.set_stemmer(Xapian::Stem("en"));
+    termgenerator.index_text("The will", 1, "S");
+    termgenerator.index_text("The will are considered stop words in xapian and "
+			     "would be thrown off, so the query I want to say "
+			     "is score, yes, score. The Score of a game is "
+			     "the determining factor of a game, the score is "
+			     "what matters at the end of the day. so my advise "
+			     "to everyone is to Score it!.", 1, "XD ");
```

Comment:
Here `"XD "` has an unwanted space in it, which will mean that the indexed terms will contain a space!